clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -728,7 +773,11 @@ class GCEUtil {
       .findAll { it.loadBalancerType == GoogleLoadBalancerType.INTERNAL }
     if (!internalLoadBalancersToAddTo) {
       log.warn("Cache call missed for internal load balancer, making a call to GCP")
-      List<ForwardingRule> projectRegionalForwardingRules = compute.forwardingRules().list(project, region).execute().getItems()
+      List<ForwardingRule> projectRegionalForwardingRules = executor.timeExecute(
+            compute.forwardingRules().list(project, region),
```

Comment:
Looks like we go back and forth between 2 and 4 spaces. Is that to match the neighbors?